packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -139,13 +139,11 @@ export function RepositoryMixin<T extends MixinTarget<Application>>(
       const options = toOptions(nameOrOptions);
       // We have an instance of
       if (dataSource instanceof juggler.DataSource) {
-        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
         const name = options.name || dataSource.name;
         const namespace = options.namespace ?? RepositoryBindings.DATASOURCES;
         const key = `${namespace}.${name}`;
         return this.bind(key).to(dataSource).tag(RepositoryTags.DATASOURCE);
       } else if (typeof dataSource === 'function') {
-        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
I think you will need to keep those `eslint-disable-next-line` comments to make `npm run lint` pass. Unfortunately, the linter is chocking at TypeScript errors caused by the fact that we don't have https://github.com/strongloop/loopback-datasource-juggler/pull/1844 yet. Feel free to wait with changes until that other PR is landed & published and loopback-next picks up the new version.